Description: Fix spelling errors in several files
Author: Marcos Fouces <marcos@debian.org>
--- a/README
+++ b/README
@@ -63,14 +63,14 @@
 	be banned for the originial ban-period, again :)
 	
      -> changed the "exempt-code" to idle=-1, so idle=0
-     	will be usefull for demonstration-purpose.	
+     	will be useful for demonstration-purpose.	
      	
      -> the information about crontab (down in the text)
 	has expired, if running this program as daemon.
 	left it, because i find the hints about crontabs
-	quite usefull.
+	quite useful.
 	
-        Maybe it is usefull to restart the program from time
+        Maybe it is useful to restart the program from time
 	to time as it might have halted due to any errors.
 	(via cron)
 
@@ -80,7 +80,7 @@
 	
      -> Make sure to reduce the size of the log-file from time
 	to time, as it is always growing when adding messages.
-	The logile or at least a part of it might be usefull
+	The logile or at least a part of it might be useful
 	to find misconfiguration or even bugs in the program.
 	
      	
@@ -108,7 +108,7 @@
 and patches.  The problem was that the original syntax of the autolog.conf
 file was a bit limiting and it was hard to work in too many additional
 features without making a big fat mess.  With the new, improved format
-of the configuration file, I should be able to accomodate many more
+of the configuration file, I should be able to accommodate many more
 future features.
 
 The configuration file consists of multiple lines, each of which describes
@@ -345,13 +345,13 @@
 Kyle Bateman
 kyle@actarg.com
 
-Be carefull when playing with a daemon. Devils like
+Be careful when playing with a daemon. Devils like
 playing, but only they know all rules of their games!
 A game, plaid with a devil becomes the devil's game.
 
 Users should be informed about such a program running.
 Especially users, which are kicked off without warning
-might get very angry when loosing several hours of work.
+might get very angry when losing several hours of work.
 Keep in mind, some are running jobs over several hours.
 
 Carsten Juerges
